docs/notes/extensions.md

```diff
@@ -65,3 +65,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 
 ### Ember.js
  - <a href="https://github.com/aomran/ember-cli-chart" target="_blank">ember-cli-chart</a>
+ 
+### Omi (v5+)
+
```

Comment:
can you remove this blank line for consistency with the other listings?

Reply:
Sure! Done~~